src/osx/Installer.Mac/uninstall.sh

```diff
@@ -10,9 +10,9 @@ then
 	exit $?
 fi
 
-# Unconfigure
+# Unconfigure (as the current user)
 echo "Unconfiguring credential helper..."
-"$GCMBIN" unconfigure
+sudo -u `logname` "$GCMBIN" unconfigure
```

Comment:
According to https://ss64.com/osx/logname.html, this is the right thing to do. It's too bad that this is a bit asymmetric from the "install" case, where we use `${USER}` instead...

Reply:
Should we be careful here?  That is, use /usr/bin/logname inside the backticks.

Reply:
Updated to the absolute path.